core/lib/dal/src/blocks_dal.rs

```diff
@@ -570,11 +566,11 @@ impl BlocksDal<'_, '_> {
             .collect::<Vec<Vec<u8>>>();
         let pubdata_input = header.pubdata_input.clone();
         let initial_bootloader_contents = serde_json::to_value(initial_bootloader_contents)
-            .map_err(|err| instrumentation.arg_error("initial_bootloader_contents", err))?;
+            .arg(&i, "initial_bootloader_contents")?;
         let used_contract_hashes = serde_json::to_value(&header.used_contract_hashes)
-            .map_err(|err| instrumentation.arg_error("header.used_contract_hashes", err))?;
-        let storage_refunds: Vec<_> = storage_refunds.iter().copied().map(i64::from).collect();
-        let pubdata_costs: Vec<_> = pubdata_costs.iter().copied().map(i64::from).collect();
+            .arg(&i, "header.used_contract_hashes")?;
+        let storage_refunds: Vec<i64> = storage_refunds.iter().map(|x| (*x).into()).collect();
+        let pubdata_costs: Vec<i64> = pubdata_costs.iter().map(|x| (*x).into()).collect();
```

Comment:
Dumb question: Is there a particular reason behind these changes? It looks like the old code would still work fine.

Reply:
There was nothing wrong. I just find it more readable this way, so I've reflexively rewritten it.